go.mod

```diff
@@ -25,7 +25,7 @@ require (
 	github.com/stretchr/testify v1.9.0
 	golang.org/x/crypto v0.28.0
 	google.golang.org/protobuf v1.35.1
-	k8s.io/apiextensions-apiserver v0.29.6
+	k8s.io/apiextensions-apiserver v0.31.1
```

Comment:
Do you know why this bump is needed? (Usually we try to keep these in-line with the client-go pkg versions our current supported Argo CD version uses)

Reply:
reverted back